PSSwagger/Definitions.psm1

```diff
@@ -894,7 +899,11 @@ function New-SwaggerSpecDefinitionCommand
         [Parameter(Mandatory=$false)]
         [AllowEmptyString()]
         [string]
-        $PSHeaderComment
+        $PSHeaderComment,
+
+        [Parameter(Mandatory=$false)]
```

Comment:
Why not use parameter set like below?  [ValidateSet('None', 'PSScriptAnalyzer')]

Reply:
For internal functions, I generally don't care about parameter validation, etc., but I can add it.